static/js/components/SourceTable.jsx

```diff
@@ -448,12 +448,12 @@ const SourceTable = ({
     return <FavoritesButton sourceID={objid} />;
   };
 
-  const renderAlias = (dataIndex) => {
-    const { id: objid, alias } = sources[dataIndex];
+  const renderOrigin = (dataIndex) => {
```

Comment:
@dmitryduev Is it fine to get rid of the Alias column here? I don't remember what it was for exactly, but it doesn't look like any sources are really using it these days and I think it's just a holdover from old iterations.

Reply:
The same physical object usually goes by multiple names in different surveys/services, that's what Alias is meant for - we just didn't get to implementing it fully. I would keep it and add origin like @guynir42 is suggesting, but otherwise it looks good to me.